### PR TITLE
Ported to Python 3.6.3 and Django 1.11

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
Project ported to django 1.11 and python 3.6.

Fixes  https://github.com/systers/vms/issues/557

# Type of Change:
- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by abhishekarya286** [comment]
Hi Anjali, Can you squash the commits?

File: vms/administrator/urls.py [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
 # local Django
 from administrator import views
 from administrator.views import GenerateReportView, AdminUpdateView, ProfileView
 
-urlpatterns = patterns(
-    '',
+urlpatterns = [
     url(r'^report/$', GenerateReportView.as_view(), name='report'),
-    url(r'^settings/$', views.settings, name='settings'),
-    url(r'^edit/(?P<admin_id>\d+)$', AdminUpdateView.as_view(), name='edit'),
```
Comment (abhishekarya286) [inline vms/administrator/urls.py]: Did you forget to add this url?

**Comment by anjali-dhanuka** [comment]
@ayushgarg1804 The first warning has been solved, but for the second one, we need to discuss the name which we'll assign in place of home.

File: vms/manage.py [hunk 2]
```diff
@@ -7,4 +7,4 @@
 
     from django.core.management import execute_from_command_line
 
-    execute_from_command_line(sys.argv)
+    execute_from_command_line(sys.argv)
```
Comment (abhishekarya286) [inline vms/manage.py]: New line at the end

File: vms/shift/urls.py [hunk 3]
```diff
@@ -1,61 +1,63 @@
...
-        ViewHoursView.as_view(),
-        name='view_hours'),
-    url(r'^view_volunteer_shifts/(?P<volunteer_id>\d+)$',
-        views.view_volunteer_shifts,
-        name='view_volunteer_shifts'),
-    url(r'^volunteer_search/$',
-        VolunteerSearchView.as_view(),
-        name='volunteer_search'),
-)
+# Django
```
Comment (abhishekarya286) [inline vms/shift/urls.py]: I think there is a problem of trailing white spaces here. You haven't changed this much lines here.

File: vms/vms/settings.py [hunk 4]
```diff
@@ -8,16 +8,14 @@
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
-BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
Comment (abhishekarya286) [inline vms/vms/settings.py]: Is this for python3?

File: vms/vms/urls.py [hunk 5]
```diff
@@ -1,12 +1,17 @@
 # Django
-from django.conf.urls import patterns, include, url
+from django.conf.urls import include, url
 from django.conf.urls.i18n import i18n_patterns
 from django.contrib import admin
 
 admin.autodiscover()
 
+
```
Comment (abhishekarya286) [inline vms/vms/urls.py]: Remove these extra lines.

File: vms/vms/urls.py [hunk 6]
```diff
@@ -1,12 +1,17 @@
...
+
+
+
 urlpatterns = i18n_patterns(
-    '',
+
```
Comment (abhishekarya286) [inline vms/vms/urls.py]: Ditto.

File: vms/administrator/urls.py (hunk 1 above)
Comment (anjali-dhanuka) [reply vms/administrator/urls.py]: I guess I haven't pulled all the code. Let me do the same.

**Comment by Monal5031** [comment]
@anjali-dhanuka I think the reason you are getting
`ModuleNotFoundError: No module named 'basePage'`
is because in python3 we have to import from same dir as
`from .basePage import BasePage` instead of `from basePage import BasePage`
Maybe try this out so as to make the existing tests pass?

**Pull request closed without merging**